repose-aggregator/src/docs/asciidoc/architecture/filter-chain.adoc

```diff
@@ -84,10 +84,8 @@ For more troubleshooting help see our <<../troubleshooting.adoc, guide>>.
 If you wish to see these numbers or exfiltrate them to another tool look into the <<../services/metrics.adoc, Metrics Service>>.
 They are listed under `org.openrepose.core.FilterProcessingTime.Delay` with an entry for each filter by name.
 
-If you are curious about the numbers for a specific request you can send in the `X-Trace-Request` header with a value of true.
-This will add a header to the response for each filter with a name of `X-_filter name_-Time` with the time it took that filter to process.
-
-It's worth noting that the reported time is inclusive of the processing time of all filter further down the chain and the origin service.
+If you are curious about the numbers for a specific request you can add the `X-Trace-Request` header to the request.
+See <<../troubleshooting.adoc#,Time Spent in Each Filter>> for more details.
```

Comment:
Is there not an anchor to directly link to on that page?

Reply:
There is, but I could not figure out how to link to it here. I did not want to link to the HTML anchor since that couples this document to the output format, but since the section title has spaces in it, I could not use that.

Actually, after playing with this a little more, I think can get it to work by switching from `URI Regex` to `_uri_regex`. That is the generated header ID, so as long as the way that header IDs are generated doesn't change, it should work. Even if that does change, the link should still be to the page, just not the specific section, which is the current behavior anyway.